shell.nix

```diff
@@ -1,77 +1,9 @@
-{
-  /* This should match Nixpkgs commit in status-mobile. */
-  source ? builtins.fetchTarball {
-    url = "https://github.com/NixOS/nixpkgs/archive/e7603eba51f2c7820c0a182c6bbb351181caa8e7.tar.gz";
-    sha256 = "sha256:0mwck8jyr74wh1b7g6nac1mxy6a0rkppz8n12andsffybsipz5jw";
-  },
-  pkgs ? import (source){
-    config = {
-      allowUnfree = true;
-      android_sdk.accept_license = true;
-    };
-    overlays = [
-      (final: prev: {
-        androidPkgs = pkgs.androidenv.composeAndroidPackages {
-          toolsVersion = "26.1.1";
-          platformToolsVersion = "33.0.3";
-          buildToolsVersions = [ "31.0.0" ];
-          platformVersions = [ "31" ];
-          cmakeVersions = [ "3.18.1" ];
-          ndkVersion = "22.1.7171670";
-          includeNDK = true;
-          includeExtras = [
-            "extras;android;m2repository"
-            "extras;google;m2repository"
-          ];
-        };
-
-        go-junit-report = prev.go-junit-report.overrideAttrs ( attrs : rec {
-          version = "2.1.0";
-
-          src = prev.fetchFromGitHub {
-            owner = "jstemmer";
-            repo = "go-junit-report";
-            rev = "v${version}";
-            sha256 = "sha256-s4XVjACmpd10C5k+P3vtcS/aWxI6UkSUPyxzLhD2vRI=";
-          };
-        });
-      })
-    ];
-  }
-}:
+# for passing build optionsm see nix/README
+# TODO complet nix/README
+{ config ? { } }:
```

Comment:
Do we even need `config` for anything? Maybe we can get rid of it entirely.

This whole file could just be:
```nix
(import ./nix).shells.default
```
And `default.nix` could just be:
```nix
import ./nix
```